xonsh/completer.py

```diff
@@ -133,7 +131,7 @@ def bash_complete(self, prefix, line, begidx, endidx):
         fnme = self.bash_complete_files.get(cmd, None)
         if func is None or fnme is None:
             return set()
-        idx = 0
+        idx = n = 0
```

Comment:
`n` is used outside of the for-loop below, and so if by some chance `splt` is length-0, then this prevents a name lookup error.  I agree that this is pretty obtuse, but it was one of the things that landscape was yelling about.  It is better and easier to 'fix' it than to suppress the error.